relay-server/src/utils/rate_limits.rs

```diff
@@ -35,6 +35,13 @@ pub fn format_rate_limits(rate_limits: &RateLimits) -> String {
 
         if let Some(ref reason_code) = rate_limit.reason_code {
             write!(header, ":{reason_code}").ok();
+        } else if !rate_limit.namespace.is_empty() {
+            write!(header, ":").ok();
```

Comment:
It took me a while to understand this code.

```suggestion
            write!(header, ":").ok(); // delimits the empty reason code
```

Reply:
Thanks, the comment helps. I was hoping to rewrite this in a better way, but everything else I could come up with ended up much more complex.